README.md

```diff
@@ -65,7 +65,7 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 <p>For Live demo click
-<a href="https://mohashyne.github.io/Module_1-Capstone-Project/"> here </a></p>
+<a href="https://mohashyne.github.io/Module1_Capstone/"> here </a></p>
 
 ## 🚀 Live Video <a name="live-video"></a>
 
```

Comment:
### LICENSE

- Your license is currently broken, let's fix this to make our readme more professional.
---

## LIVE DEMO

- The live demo version seems different from your main live version. kindly update this. :pray: 
---

## CONTRIBUTING

- Please, kindly add the link to the issues page of this current project so users can easily navigate to the issues page.

Reply:
changes applied and links are now working.